docs/secure-connections/sauce-connect-5/installation.md

```diff
@@ -1,217 +1,12 @@
 ---
 id: installation
-title: Downloading Sauce Connect Proxy 5
-sidebar_label: Install
+title: Install Sauce Connect 
+sidebar_label: Installation
 ---
 
-import useBaseUrl from '@docusaurus/useBaseUrl';
-import ScTable from '../../../src/components/scTable.jsx';
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-
-:::caution
-See [this document](/secure-connections/sauce-connect/installation/) for the most recent Sauce Connect 4.x.x version download instructions.
-:::
-
-This topic describes how to install Sauce Connect Proxy version 5 to your machine.
-
-## What You’ll Need
-
-- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
-
-## Downloading Sauce Connect Proxy
```

Comment:
there should be links to download binaries downloads, downloading a binary is still a legitimate way to install sc. I think that each OS, should have a subset of this table

Reply:
All the links are still there. They can be found in specific instructions, e.g.
<img width="1024" alt="Screenshot 2024-02-19 at 08 32 04" src="https://github.com/saucelabs/sauce-docs/assets/57443889/e3037cab-74d5-4844-97ca-50514c4376d4">

Reply:
I think the table containing links is useful - you can click on the link in any browser and the binary gets downloaded.